### PR TITLE
Adding KS Technologies KST5420 - Impulse Proof of Coverage Test Tool

### PR DESCRIPTION
**Pull request by Devin-Saavedra**
#### Summary
Adding KS Technologies KST5420 - Impulse Proof of Coverage Test Tool Device.

#### Changes
Initial Add of KS Technologies and the KST5420 Product.

### CODE REVIEW

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @Devin-Saavedra thank you for adding KST devices, there is a branch conflict, can you take a look to it.

File: vendor/ks-technologies/kst5420-profile.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+vendorProfileID: 800
```
[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/ks-technologies/kst5420-profile.yaml]: @LDannijs Gotcha, I have updated this. Just so I am clear here, does this vendorProfileID need to be different per product?

[the remark this answers is not in the document]
Comment (LDannijs) [reply vendor/ks-technologies/kst5420-profile.yaml]: Yes it should be different per product. It's up to your own choice on how to label them, as long as each device has a unique ID.

[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/ks-technologies/kst5420-profile.yaml]: Perfect, thank you for the explanation!

I thought I fixed the conflict in the index.yaml file but apparently not. Am I not supposed to update that file with our company info?

[the remark this answers is not in the document]
Comment (LDannijs) [reply vendor/ks-technologies/kst5420-profile.yaml]: What conflict is it giving you? I also see you've edited some info from other vendors, is that correct?

[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/ks-technologies/kst5420-profile.yaml]: I thought I had pulled in the most recent from master and just added in my companies info to the very bottom, so I did not mean to edit any other vendors info. 

If I click on the Resolve conflicts button in the Changes approved section below it is just showing me the addition of my companies info to the index.yaml file.

[the remark this answers is not in the document]
Comment (LDannijs) [reply vendor/ks-technologies/kst5420-profile.yaml]: Oh i see. I'll have my colleague @Jaime-Trinidad look at it.

**Review by Jaime-Trinidad — APPROVED** [review]

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by LDannijs** [comment]
So me and @Jaime-Trinidad took a look at your PR and it seems some vendor folders were missing and it was giving errors. We suggest you close this PR, and make sure your fork is fully up-to-date with the master before adding your own files and opening a new PR. Apologies for this inconvenience.

**Comment by Devin-Saavedra** [comment]
No problem, I will make sure I have the very latest before submitting a PR. Thank you for taking the time to take a look at the issues.

**Pull request closed without merging**